http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -192,14 +191,20 @@ object Boot extends IOApp with LazyLogging {
       httpClientWithCustomSSL <- blaze.BlazeClientBuilder[F](blockingEc, Some(sslContext)).resource
       clientWithRetryWithCustomSSL = Retry(retryPolicy)(httpClientWithCustomSSL)
       clientWithRetryAndLogging = Http4sLogger[F](logHeaders = true, logBody = false)(clientWithRetryWithCustomSSL)
+      clientWithNoRetryNoHeaderLogging = Http4sLogger[F](logHeaders = false, logBody = false)(httpClientWithCustomSSL)
+      // TODO: Use this client without retry for RStudioDAO because `checkClusterStatus` in `ClusterToolMonitor` will check
```

Comment:
Ah good catch, yeah we should fix this (can be in a separate PR).

Reply:
I guess RStudio-enabled clusters will have the same problem, but with Jupyter. Maybe I'll put in a PR for this.

Reply:
that would be great...I tried to do it...but it wasn't trivial..so just putting a todo here for now